### ISSUE TITLE
Why can't I login and use skin in my private server?

### ISSUE THREAD

**Issue by JoeLiu333** [OgarProject/Ogar#460]
I only can "play as guest" in my private server.But I see the youtuber 'ArcadeGo' can use skin in his server,I really want to do it.How can I do that?

**Comment by Andrews54757** [comment]
Ogar doesnt support skins. but Ogar unlimited does. Go to ogar unlimited for skins. Just search it up on google

**Comment by NatsuTheGreat** [comment]
Arcade go uses agar.iomods, that extension lets him use his skin.

**Comment by Agarian** [comment]
https://github.com/AJS-development/Ogar-unlimited

**Comment by NatsuTheGreat** [comment]
@vram4 Close?

**Comment by PrinceDark9243** [comment]
Yeah Use Ogar Unlimited! Its 100% More Better than Ogar Master!

**Comment by NatsuTheGreat** [comment]
Yeah with all the errors going around. So much better, @PrinceDark9243

**Comment by Andrews54757** [comment]
@NatsuTheGreat Im working with all the errors, Now the AJS dev team has 5 members that edit so we can fix bugs faster

**Comment by NatsuTheGreat** [comment]
@Andrews54757 I meant no offense to you :P

**Comment by Agarian** [comment]
lel
